STYLE.md

```diff
@@ -0,0 +1,228 @@
+# Introduction
+All code needs to be formatted according to .pre-commit-config.yaml. Each pull request is checked by [Formatting Tests](https://github.com/Azure/azure-osconfig/blob/dev/.github/workflows/formatting.yml).
```

Comment:
>All code needs to be formatted according to .pre-commit-config.yaml [](http://example.com/codeflow?start=0&length=67)

pre-commit does not format for the coding style. Only does additional and separate clean-up. Having it here sounds misleading as it makes the reader think that by running it this coding style will be automatically applied. Recommend to move it from STYLE back to CONTRIBUTING where it was.